src/SFA.DAS.EmployerFinance/Commands/RefreshPaymentData/RefreshPaymentDataCommandHandler.cs

```diff
@@ -56,25 +56,29 @@ protected override async Task HandleCore(RefreshPaymentDataCommand message)
 
             try
             {
-                _logger.Info($"GetAccountPayments for AccountId = '{message.AccountId}' and PeriodEnd = '{message.PeriodEnd}'");
+                _logger.Info($"GetAccountPayments for AccountId = '{message.AccountId}' and PeriodEnd = '{message.PeriodEnd}' CorrelationId: {message.CorrelationId}");
 
-                payments = await _paymentService.GetAccountPayments(message.PeriodEnd, message.AccountId);
+                payments = await _paymentService.GetAccountPayments(message.PeriodEnd, message.AccountId, message.CorrelationId);
             }
             catch (WebException ex)
             {
-                _logger.Error(ex, $"Unable to get payment information for AccountId = '{message.AccountId}' and PeriodEnd = '{message.PeriodEnd}'");
+                _logger.Error(ex, $"Unable to get payment information for AccountId = '{message.AccountId}' and PeriodEnd = '{message.PeriodEnd}' CorrelationId: {message.CorrelationId}");
             }
 
             if (payments == null || !payments.Any())
             {
-                _logger.Info($"GetAccountPayments did not find any payments for AccountId = '{message.AccountId}' and PeriodEnd = '{message.PeriodEnd}'");
+                _logger.Info($"GetAccountPayments did not find any payments for AccountId = '{message.AccountId}' and PeriodEnd = '{message.PeriodEnd}' CorrelationId: {message.CorrelationId}");
 
                 await PublishRefreshPaymentDataCompletedEvent(message, false);
 
                 return;
             }
 
-            _logger.Info($"GetAccountPaymentIds for AccountId = '{message.AccountId}' and PeriodEnd = '{message.PeriodEnd}'");
+            _logger.Info($"GetAccountPaymentIds for AccountId = '{message.AccountId}' and PeriodEnd = '{message.PeriodEnd}' CorrelationId: {message.CorrelationId}");
+
+            var failingPayment = payments.Where(p => p.ApprenticeshipId == 743445).ToList();
+
+            _logger.Info($"Got payment for appId: {failingPayment.FirstOrDefault()?.ApprenticeshipId}, paymentIds: [{string.Join("'", failingPayment.Select(x => x.Id))}] and PeriodEnd = '{message.PeriodEnd}' CorrelationId: {message.CorrelationId}");
```

Comment:
I think this bit can be removed as this was for a specific investigation back in January